tests/broker-engine/cma.robot

```diff
@@ -0,0 +1,361 @@
+    
+    ${echo_command}   Ctn Echo Command  "OK check2 - 127.0.0.1: rta 0,010ms, lost 0%|rta=0,010ms;200,000;500,000;0; pl=0%;40;80;; rtmax=0,035ms;;;; rtmin=0,003ms;;;;"
+    Ctn Engine Config Add Command  ${0}    otel_check_icmp_2  ${echo_command}    OTEL connector
+
+    #update conf engine, it must be taken into account by agent
+    Log To Console    modify engine conf and reload engine
+    Ctn Reload Engine
+
+    #wait for new data from agent
+    ${start}    Ctn Get Round Current Date
+    ${content}    Create List    description: \"OK check2
+    ${result}    Ctn Find In Log With Timeout    ${engineLog0}    ${start}    ${content}    22
+    Should Be True    ${result}    "description: "OK check2" should be available.
+
+    ${result}    Ctn Check Host Check Status With Timeout    host_1    30    ${start}    0    OK check2 - 127.0.0.1: rta 0,010ms, lost 0%
+    Should Be True    ${result}    hosts table not updated
+
+
+BEOTEL_CENTREON_AGENT_CHECK_SERVICE
+    [Documentation]    agent check service and we expect to get it in check result
+    [Tags]    broker    engine    opentelemetry    MON-63843
+    Ctn Config Engine    ${1}    ${2}    ${2}
+    Ctn Add Otl ServerModule
+    ...    0
+    ...    {"otel_server":{"host": "0.0.0.0","port": 4317},"max_length_grpc_log":0,"centreon_agent":{"check_interval":10, "export_period":15}}
+    Ctn Config Add Otl Connector
+    ...    0
+    ...    OTEL connector
+    ...    opentelemetry --processor=centreon_agent --extractor=attributes --host_path=resource_metrics.resource.attributes.host.name --service_path=resource_metrics.resource.attributes.service.name
+    Ctn Engine Config Replace Value In Services    ${0}    service_1    check_command    otel_check
+    Ctn Set Services Passive       0    service_1
+
+    ${check_cmd}  Ctn Check Pl Command   --id 456
+
+    Ctn Engine Config Add Command    ${0}    otel_check   ${check_cmd}    OTEL connector
+
+    Ctn Engine Config Set Value    0    log_level_checks    trace
+
+    #service_1 check fail CRITICAL
+    Ctn Set Command Status    456    ${2}
+
+    Ctn Config Broker    central
+    Ctn Config Broker    module
+    Ctn Config Broker    rrd
+    Ctn Config Centreon Agent
+    Ctn Broker Config Log    central    sql    trace
+
+    Ctn Config BBDO3    1
+    Ctn Clear Retention
+
+    ${start}    Ctn Get Round Current Date
+    ${start_int}    Ctn Get Round Current Date
+    Ctn Start Broker
+    Ctn Start Engine
+    Ctn Start Agent
+
+    # Let's wait for the otel server start
+    ${content}    Create List    unencrypted server listening on 0.0.0.0:4317
+    ${result}    Ctn Find In Log With Timeout    ${engineLog0}    ${start}    ${content}    10
+    Should Be True    ${result}    "unencrypted server listening on 0.0.0.0:4317" should be available.
+    
+    ${result}    Ctn Check Service Check Status With Timeout    host_1  service_1  60  ${start_int}  2  Test check 456
```

Comment:
4 spaces not respected.

Reply:
And also same remark as for hosts, it looks more relevant to check resources table as it will be the only one to work "soon".